src/comment_worker.py

```diff
@@ -283,6 +283,9 @@ def invest(self, sess, comment, investor, amount, suffix):
         if amount < minim or amount < 100:
             return comment.reply_wrap(message.modify_min_invest(minim))
 
+        if amount > 1e9:
+            return comment.reply_wrap("Investing limit is 1 billion")
```

Comment:
Wouldn't it be better to accept the investment but cap it?

`if amount > 1e9:
    amount = 1e9
`

Reply:
It probably should provide some notice, though.

Reply:
maybe it could do both?  accept it and add the comment?